### DISCUSSION TITLE
Call for Collaborators: Join Our Project and Help Shape Its Future!

### DISCUSSION THREAD

**Discussion by areed1192** [areed1192/python-trading-robot#32] [Announcements]
Hello everyone! I'm excited to reach out and inquire if there are any individuals interested in lending their expertise to contribute to this project. We are seeking a team of dedicated collaborators who can actively participate in managing the repository, handling pull requests, making valuable contributions, and providing overall project management support. While I will certainly continue to offer my assistance, it is vital to ensure the project's ongoing availability for all users. Your involvement would be greatly appreciated!

**Comment by JacobChrist** [comment]
Is the Ameritrade API going to continue to be available with Schwab?

**Reply by areed1192** [reply]
From everything I've seen so far, yes it will still be available. However, I would imagine URLs would change, and some endpoints may be modified/removed/added.

**Reply by BDSoftley** [reply]
So far Schwab seems to be slow with information, My hope is they will just convert the endpoints and leave the rest alone.

**Reply by areed1192** [reply]
So, they sent out an email a few days ago going over the changes, I've had a few people try to get access to the API for their personal accounts but no luck so far. I have a feeling personal usage of the API will not be the focus until they get all the commerical customers taken care of first.
